### PR TITLE
added solution for Custom Configuration

### PR DESCRIPTION
**Pull request by jamasten**
Per a request from @webreidi I'm adding my code for the AVD Post Update Custom Config as an example for the public to consume from the official AVD repository.

### CODE REVIEW

File: wvd-sh/arm-template-customization/readme.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Azure Virtual Desktop - Post Update Custom Configuration solution
```
Comment (dhrump) [inline wvd-sh/arm-template-customization/readme.md:1]: We should change the name to custom configuration

File: wvd-sh/arm-template-customization/script.ps1 [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿##############################################################
+#  Run the Virtual Desktop Optimization Tool (VDOT)
+##############################################################
+# https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool
+
+# Download VDOT
+$URL = 'https://github.com/The-Virtual-Desktop-Team/Virtual-Desktop-Optimization-Tool/archive/refs/heads/main.zip'
```
Comment (dhrump) [inline wvd-sh/arm-template-customization/script.ps1:7]: This script is not valid since paths are not working. We should add here more workable script

Comment (jamasten) [reply wvd-sh/arm-template-customization/script.ps1:7]: @dhrump the feature in the portal is called "Post Update Custom Configuration".  I believe it makes sense to at least keep that text so customers can easily identify the solution.  The VDOT path works for everyone else I've shared my code with, internally and externally.  Not sure why you're having a problem.  VDOT is a common post configuration with AVD and I would hate to see the code updated with a solution that isn't helpful.

File: wvd-sh/arm-template-customization/readme.md (hunk 1 above)
Comment (jamasten) [reply wvd-sh/arm-template-customization/readme.md:1]: @dhrump the feature in the portal is called "Post Update Custom Configuration".  I believe it makes sense to at least keep that text so customers can easily identify the solution.

File: wvd-sh/arm-template-customization/script.ps1 (hunk 2 above)
Comment (dhrump) [reply wvd-sh/arm-template-customization/script.ps1:7]: interesting. i was not able to access the path

File: wvd-sh/arm-template-customization/readme.md (hunk 1 above)
Comment (dhrump) [reply wvd-sh/arm-template-customization/readme.md:1]: We are changing name internally. I will update the changes and push our the PR

**Pull request merged**